inst/tests/tests.Rraw

```diff
@@ -18315,3 +18315,45 @@ if (test_bit64) {
   apple = data.table(id = c("a", "b", "b"), time = c(1L, 1L, 2L), y = i64v[1:3])
   test(2248, dcast(apple, id ~ time, value.var = "y"), data.table(id = c('a', 'b'), `1` = i64v[1:2], `2` = i64v[4:3], key='id'))
 }
+
+# trunc.char misbehaves for multibyte characters, #5096
+# Tests for combining characters
+options(datatable.prettyprint.char=4L)
+accented_a = "\u0061\u0301"
```

Comment:
@joshhwuu PTAL at this approach, I hope it improves readability but I would like a second pair of eyes to agree.

Reply:
I like this, makes it easier to understand.